cobl2/templates/cognateset/detail_html.mako

```diff
@@ -1,9 +1,9 @@
 <%inherit file="../${context.get('request').registry.settings.get('clld.app_template', 'app.mako')}"/>
 <%namespace name="util" file="../util.mako"/>
 <%! active_menu_item = "cognatesets" %>
-<%block name="title">${_('Cognateset')} ${ctx.name}</%block>
+<%block name="title">${_('Cognate Set')} ${ctx.name}</%block>
```

Comment:
Did translation not work? Putting the exact string you want in the UI into `_('...')` can only make things worse, btw.